mmdet/models/backbones/res2net.py

```diff
@@ -0,0 +1,503 @@
+import math
+
+import torch
+import torch.nn as nn
+import torch.utils.checkpoint as cp
+import torch.utils.model_zoo as model_zoo
+from mmcv.cnn import (build_conv_layer, build_norm_layer, constant_init,
+                      kaiming_init)
+from torch.nn.modules.batchnorm import _BatchNorm
+
+from mmdet.ops import build_plugin_layer
+from ..builder import BACKBONES
+from .resnet import ResNet
+
+
+class Bottle2neck(nn.Module):
```

Comment:
We may inherit from `Bottleneck` in `ResNet` so that part of initialization could be saved.

Reply:
The bottle2neck is now inherited from Bottleneck